lib/src/spec/constants.dart

```diff
@@ -0,0 +1,8 @@
+class LogLevel{
+  static final int NONE = 99;
+  static final int VERBOSE = 2;
+  static final int DEBUG = 3;
+  static final int INFO = 4;
+  static final int WARN = 5;
+  static final int ERROR = 6;
```

Comment:
Is it idiomatic to have LogLevel with `NONE` as opposed to `None? 
What do Dart developers prefer / use normally?

Reply:
Matt's got a valid point here and it raises something else I should have spotted, sorry!

First, yes, we should be aiming to be as Dart-idiomatic as possible and that means adhering to [Effective Dart](https://dart.dev/guides/language/effective-dart) wherever possible - thus these, as 'constant names' / 'constant variables' should [`PREFER` using `lowerCamelCase`](https://dart.dev/guides/language/effective-dart/style#prefer-using-lowercamelcase-for-constant-names) - i.e. `none`, `verbose`, `debug`, etc..

Secondly, I think you can and should be using `const` rather than `final` as these values can be 'calculated' at compile time. See [this answer](https://stackoverflow.com/a/50431087/392847).

Reply:
Updated in 822e6407feea365d548a0282b15880065f798e6f